blockprod/src/detail/job_manager/jobs_container.rs

```diff
@@ -76,10 +76,7 @@ impl JobsContainer {
     /// Returns true if the job was removed, false if it was not found.
     fn remove_job(&mut self, job_key: JobKey, and_stop: bool) -> bool {
         match self.jobs.entry(job_key) {
-            Entry::Vacant(j) => {
-                log::error!("Attempted to stop non-existent job: {j:?}");
-                false
-            }
+            Entry::Vacant(_) => false,
```

Comment:
Could you please elaborate on the rationale behind this change? Will it not just mask some sort of job management issue?

Reply:
No problem. I've updated the description.